cpp/tests/copying/split_tests.cpp

```diff
@@ -905,6 +905,8 @@ void split_structs_no_children(SplitFunc Split, CompareFunc Compare)
 {
   using namespace cudf::test;
 
+  auto const TWO = static_cast<std::size_t>(2);
```

Comment:
Could we use 2ul instead?

Reply:
:+1: yea, i agree - literal is better. Good catch.